docs/csharp/programming-guide/concepts/linq/join-operations.md

```diff
@@ -21,6 +21,20 @@ A *join* of two data sources is the association of objects in one data source wi
 |Join|Joins two sequences based on key selector functions and extracts pairs of values.|`join … in … on … equals …`|<xref:System.Linq.Enumerable.Join%2A?displayProperty=nameWithType><br /><br /> <xref:System.Linq.Queryable.Join%2A?displayProperty=nameWithType>|  
 |GroupJoin|Joins two sequences based on key selector functions and groups the resulting matches for each element.|`join … in … on … equals … into …`|<xref:System.Linq.Enumerable.GroupJoin%2A?displayProperty=nameWithType><br /><br /> <xref:System.Linq.Queryable.GroupJoin%2A?displayProperty=nameWithType>|  
   
+## Query Expression Syntax Examples  
+  
+### Join  
```

Comment:
Should localization be avoided here ?

Reply:
Yes, we should avoid localization.  @JasonCard What's the best way to ensure that in an H3 heading?

Reply:
@BillWagner You can use the no-loc metadata mentioned [here](https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#non-localized-strings), for example:
---
title: Title of the Document
author: author-name
description: Description for the document
no-loc: [Join, GroupJoin]
---

Reply:
@SuperNova911 Can you add the line referenced above in Jason's comment? Then, this is ready to :shipit: